testing/test_assertrewrite.py

```diff
@@ -279,6 +279,17 @@ def f():
             assert False or 4 % 2
         assert getmsg(f) == "assert (False or (4 % 2))"
 
+    @pytest.mark.skipif("sys.version_info < (3,5)")
+    def test_at_operator_issue1290(self):
+        def f():
+            class Matrix:
```

Comment:
I guess you will have to use `eval` or `exec` here to avoid syntax errors in Python versions < 3.5. :grin: